src/main/webapp/transcript.js

```diff
@@ -1,9 +1,84 @@
+
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const TRANSCRIPT_CONTAINER = 'transcript-container';
+const ENDPOINT_TRANSCRIPT = '/transcript';
+
 /**
  * Sends a POST request to the transcript.
  */
 function sendPostToTranscript() {
-  // TODO: Update with user input.
-  const testParamsString = 'id=123456789&video=8PrOp9t0PyQ';
-  const params = new URLSearchParams(testParamsString);
-  fetch('/transcript', {method: 'POST', body: params});
+  // TODO: Update with user input and URL Builder.
+  const paramsString = 'id=123456789&video=3ymwOvzhwHs';
```

Comment:
paramsString is not very descriptive or what this id is for? Is it transcriptUrl?

Reply:
The string is the query parameters for a certain lecture recording. I updated the variable name to be `lectureQueryString` so hopefully it's clearer.